libevm/go.mod

```diff
@@ -0,0 +1,31 @@
+module libevm
+
+go 1.18
+
+require github.com/ethereum/go-ethereum v1.10.16
```

Comment:
We should double check which exact version of go-ethereum we want to use. So far the version was just chosen as it was the latest one available when development started.